pingdom-checks/patient-generated-data.ping

```diff
@@ -0,0 +1,13 @@
+# -*- sh -*-
+
+PGD_SLACK_ID=PLACEHOLDER
```

Comment:
what's this?

Reply:
Oh crap, that is something I put as a reminder to ask a question, and after I asked it I forgot to remove it. Well, that is embarrassing.